cmd/csi-snapshotter/main.go

```diff
@@ -134,23 +138,25 @@ func main() {
 	defer cancel()
 
 	// Find driver name
-	*snapshotter, err = csiConn.GetDriverName(ctx)
+	*snapshotter, err = csirpc.GetDriverName(ctx, csiConn)
 	if err != nil {
-		glog.Error(err.Error())
+		glog.Errorf("error getting CSI driver name: %v", err)
 		os.Exit(1)
 	}
+
 	glog.V(2).Infof("CSI driver name: %q", *snapshotter)
 
 	// Check it's ready
-	if err = waitForDriverReady(csiConn, *connectionTimeout); err != nil {
-		glog.Error(err.Error())
+	if err = csirpc.ProbeForever(csiConn, csiTimeout); err != nil {
```

Comment:
Why is ProbeForever used here while in external-provisioner Probe is used?

Reply:
I haven't updated external-provisioner yet to use csi lib utils, but I'll update to use ProbeForever when I get to it :)